### PR TITLE
Feat/replace colorway #163

### LINKED ISSUE #163: Replace UI Colorway
**Blocked by UI package.**

## Current Behavior:
- Dark color way 

## Expected Behavior:
- Light color way to match the [Frequency Style Guidelines](https://www.figma.com/design/EjFt1XdacOogDZoQ7RetrI/Frequency-Style-Guidelines?node-id=25-32&t=pFC8eElSnqm40ms5-0). As seen on:
  -  https://docs.frequency.xyz/
  - https://www.frequency.xyz/

## Acceptance Criteria:
Use the UI package to implement the following:

- [ ] Body Background color -> light tan
- [ ] Remove the background image
- [ ] Info Sections & Nav Background color -> tan
- [ ] Title/Header Text colors -> dark blue
- [ ] Body Text colors -> black
- [ ] Button colors -> teal/white text
- [ ] Link colors (default and hover) -> teal -> bright blue
- [ ] Dropdown colors -> teal/white text

## Dev Notes:
If there is anything missing from the list above, feel free to use your best judgement on color updates.

### PR DESCRIPTION
**Pull request by shannonwells**
## Goal

The goal of this PR is to update the colors for Provider Dashboard and make any necessary other adjustments.

Closes #163 

### Cehcklist
- [x] PR Self-Review and Commenting

with @claireolmstead

### CODE REVIEW

File: src/components/AddKeyRequirements.svelte [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <div>
   <div class="label mb-2">Requirements</div>
-  <ol class="ordered-list text-xs">
+  `
```
Comment (claireolmstead) [inline src/components/AddKeyRequirements.svelte:3]:
```suggestion
```

Comment (shannonwells) [reply src/components/AddKeyRequirements.svelte]: fixed

File: src/components/NavItem.svelte [hunk 2]
```diff
@@ -11,7 +11,7 @@
   {href}
   on:click={onClick}
   class={` flex h-[100px] items-center justify-center text-sm font-bold ${
-    isActive && 'bg-bg-black-active shadow-blue-border'
+    isActive && 'bg-cream text-teal shadow-blue-border'
```
Comment (claireolmstead) [inline src/components/NavItem.svelte:14]: Didn't look quite right to me, but I think this is better. Changing the hex in "shadow-blue-border" to be the bright blue too.
```suggestion
    isActive && 'bg-cream text-bright-blue shadow-blue-border'
```

File: tailwind.config.js [hunk 3]
```diff
@@ -11,20 +11,26 @@ export default {
...
       divider: '#504D4D',
       overlay: 'rgba(57, 52, 52, 0.75)',
-      error: '#E70F0F',
     },
     width: {
-      modal: '420px',
+      modal: '480px',
       'single-block': '550px',
       dashboard: '1010px',
       full: '100%',
```
Comment (claireolmstead) [inline tailwind.config.js:36]: It won't let me suggest that far down, but change

'blue-border': '8px 0 0 #69B9CD',
to 
'blue-border': '8px 0 0 #5E69FF',

**Review by claireolmstead — APPROVED** [review]
- Paired
- Tested code
- Compared to Figma

Looks great!!

**Pull request merged**